test/alternator/test_tablets.py

```diff
@@ -118,14 +113,9 @@ def test_alternator_tablets_and_lwt(dynamodb):
         'AttributeDefinitions': [ { 'AttributeName': 'p', 'AttributeType': 'S' }]}
     with new_test_table(dynamodb, **schema) as table:
         assert uses_tablets(dynamodb, table)
-        # This put_item should pass after #18068 is fixed:
-        with pytest.raises(ClientError, match="InternalServerError"):
-            table.put_item(Item={'p': 'hello'})
-            assert table.get_item(Key={'p': 'hello'})['Item'] == {'p': 'hello'}
+        table.put_item(Item={'p': 'hello'})
+        assert table.get_item(Key={'p': 'hello'})['Item'] == {'p': 'hello'}
 
-# An Alternator table created tablets and with a write isolation
-# mode that doesn't use LWT ("forbid_rmw") works normally, even
-# before #18068 is fixed.
```

Comment:
There was no reason to remove this comment, which explains well why this test exists...

Reply:
I honestly can't imagine a situation where this comment adds value. The information it conveys can be deduced from the test name and the use of 'forbid_rmw'. However, this is not the point of this commit, I'll return this comment back.